src/row.jl

```diff
@@ -131,3 +136,54 @@ Lighthouse <v0.14.0).
 function _evaluation_row_dict(row::EvaluationRow)
     return Dict(string(k) => v for (k, v) in pairs(NamedTuple(row)) if !ismissing(v))
 end
+
+#####
+##### `ObservationRow`
+#####
+
+# Redefinition is workaround for https://github.com/beacon-biosignals/Legolas.jl/issues/9
+const OBSERVATION_ROW_SCHEMA = Legolas.Schema("lighthouse.observation@1")
+"""
+    const ObservationRow = Legolas.@row("lighthouse.observation@1",
+                                        predicted_hard_label::Int64,
+                                        predicted_soft_labels::Vector{Float32},
+                                        elected_hard_label::Int64,
+                                        votes::Union{Missing,Vector{Int64}})
+
+A type alias for [`Legolas.Row{typeof(Legolas.Schema("lighthouse.observation@1"))}`](https://beacon-biosignals.github.io/Legolas.jl/stable/#Legolas.@row)
+representing the per-observation input values required to compute [`evaluation_metrics_row`](@ref).
+"""
+const ObservationRow = Legolas.@row("lighthouse.observation@1",
+                                    predicted_hard_label::Int64,
+                                    predicted_soft_labels::Vector{Float32},
+                                    elected_hard_label::Int64,
+                                    votes::Union{Missing,Vector{Int64}})
+
+function _observation_table_to_inputs(observation_table)
+    Legolas.validate(observation_table, OBSERVATION_ROW_SCHEMA)
+    df_table = DataFrame(observation_table; copycols=false)
```

Comment:
I think we don't actually need a `DataFrame`, just a column table:
```suggestion
    df_table = Tables.columns(observation_table)
```
(DataFrame additionally supports `transform`, `select`, etc, but here we just want to access columns)